docs/modules/operation/pages/database-reports/templates/standard.adoc

```diff
@@ -1,8 +1,9 @@
 
```

Comment:
Rename this file to "introduction.adoc" for use in the navigation tree as a dedicated page instead of including it on each individual report.